src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate: function(callback, number) {
+      this.result = callback(this.result, number);
```

Comment:
Consider binding context to use this.result inside 'add' and other functions

Reply:
Also, create a check if the callback argument is valid and it is a function to prevent further errors

Reply:
32ffa75571eb4adcda1ba963d24b5f7cf5ef3cdf
I changed the code, please check